source/sonata_tech.rst

```diff
@@ -576,3 +576,23 @@ Required fields for ``Spykfunc``:
 .. _specification: https://github.com/AllenInstitute/sonata/blob/master/docs/SONATA_DEVELOPER_GUIDE.md
 .. _enumeration: https://github.com/AllenInstitute/sonata/blob/master/docs/SONATA_DEVELOPER_GUIDE.md#nodes---enum-datatypes
 .. _Tsodyks Markram Model: https://www.pnas.org/content/94/2/719
+
+Format of the electrodes_file
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The format of the weights file is described below:
+
+.. table::
+
+  ===================================== ======================= ========== =================== ============= ==========================================
+  Group                                 Field                   Type       Shape               Requirement   Description
+  ===================================== ======================= ========== =================== ============= ==========================================
+  /electrodes/contacts/{electrodename}  layer                   utf8       1                   Optional      Layer of the circuit in which {electrodename} is located. If the electrode is in a region without cortical layers, then "NA". If the electrode is outside the brain, then "Outside"
```

Comment:
What is the meaning of `contacts`? Everything under this dataset appears to be metadata, I think?

Reply:
Yes, it is all metadata

Reply:
Why is it called `contacts`?

Reply:
Originally, the intent was to have each {electrodename} refer to a different contact of the same electrode array. Now that we allow recordings from multiple arrays, this has been changed to /electrodes/{array_name}/{electrodename}